lib/chef/knife/bmcs_server_show.rb

```diff
@@ -103,10 +38,10 @@ def run
             raise unless service_error.serviceCode == 'NotAuthorizedOrNotFound'
           else
             add_vnic_details(vnic_info.data)
-            # for now, only display information for primary vnic
             if vnic_info.data.is_primary == true
+              vnic_array.unshift(vnic_info.data) # make primary interface first in the array
+            else
```

Comment:
Does this now show secondary vnics? Let's put that in another commit so that we can merge this now.